README.md

```diff
@@ -113,3 +113,17 @@ Available function:
 ## Further reading
 
 More detailed documentation can be found in the [docs section](https://github.com/redhatinsights/insights-chrome/tree/master/docs)
+
+## Staleness
+
+A bot will post a comment after 60 days of inactivity giving the opener 5 days to update their issue/PR before it's closed.
```

Comment:
@ryelo FWIW this probably makes more sense to add to a contribution guide over the readme. Maybe in the future we can create one.

Reply:
I can whip one up real quick and add it to this PR